cloudformation/cloudformation.yaml

```diff
@@ -588,4 +590,22 @@ Resources:
       EvaluationPeriods: 1
       Threshold: 1
       ComparisonOperator: LessThanThreshold
-    DependsOn: fulfilmentCheckerMetricFilter
\ No newline at end of file
+    DependsOn: fulfilmentCheckerMetricFilter
+
+  ExecutionFailureAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmName: !Sub State Machine Failure in ${Stage} (Fulfilment state machine)
```

Comment:
Don't we need an AlarmActions here to make this useful?

Reply:
yeah I left that part out when I copied your PR and then forgot about it.
I see you have an sns endpoint for support-workers, should we do the same for this? I'm not sure how that works. is it attached to a mailing list so everyone gets an email when it enters alarm state?

Reply:
Yes that's right - we should probably have an SNS topic configured to email the devs in our team (do we already have a Google group for this that we can use?)